_posts/jpanken/2017-05-22-jaffa-turtle-hack.md

```diff
@@ -0,0 +1,19 @@
+--- 
+layout: post
+author: jpanken
+title: "Jaffa's Turtle Exercise"
+---
+Here's the program I'm embedding:
+<iframe src="https://trinket.io/embed/python/fb814f79b7" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+
+Careful, my turtle program may induce seizures...Some people find it captivating, others nauseating.  Unfortunately, I'm one of the latter.
+
+
+I started writing this program by Googling "python turtle tricks."  This pulled up some optical illusions based on loops that I tested.  After several trippy results, I found a design that looked interesting (https://michael0x2a.com/blog/turtle-examples).  After observing how the loop works, I was able to modify the lengths and colors to produce something that looked like a sun.  After playing around more, I saw that I could turn it into an eye as well.  Then, I thought about putting a star in the middle of the iris to suggest "starry eyed."  
```

Comment:
I like how you change backgrounds!  Feels like crashing lightning.  I'm also glad that you modified the code you borrowed extensively, and cited it.  That's exactly what I'm looking for if you use external sites for inspiration.